packages/ecr-credential-provider-1.32/ecr-credential-provider-1.32.spec

```diff
@@ -0,0 +1,77 @@
+%global goproject github.com/kubernetes
+%global gorepo cloud-provider-aws
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.32.0
+%global rpmver %{gover}
+
+%global _dwz_low_mem_die_limit 0
+
+Name: %{_cross_os}ecr-credential-provider-1.32
+Version: %{rpmver}
+Release: 0.beta1%{dist}
```

Comment:
Can we run `rpm -qpi ` on one of these rpms to verify the versioning looks good? Let's add this to the PR testing description.